TypeScript.YAML-tmLanguage

```diff
@@ -169,7 +160,7 @@ repository:
       '5': { name: keyword.operator.ts }
       '6': { name: entity.name.function.ts }
       '7': { name: keyword.operator.ts }
-    end: '(?=\}|;|,|$)|(?<=\})'
```

Comment:
This change because the current definition breaks the following case:
```ts
class Test {
	acid(trip: string | {}, buttweasel: string) 
	{ // here
	}
}
```
because it thought the method definition ends at new line